### DISCUSSION TITLE
can we import PDF content to docfx

### DISCUSSION THREAD

**Discussion by sandaruwan4ever** [dotnet/docfx#7961] [Q&A]
Hello Team,
I have seen that we can export pdf files ,but can we import pdf/doc files or contents to docfx in an easy way? Like the whole document at once ?

**Comment by KalleOlaviNiemitalo** [comment]
You can have pdf/doc files as content so that DocFX copies them to the output directory and you can link to them. But DocFX cannot translate them to HTML or display them together with the Table of Contents of the site.

**Reply by sandaruwan4ever** [reply]
So, we have to manually copy/paste and create the structure of the site accordingly.
Ok, thanks for the quick reply.
